openmdao/docs/openmdao_book/features/building_blocks/components/ks_comp.ipynb

```diff
@@ -394,6 +394,150 @@
    "source": [
     "assert_near_equal(prob.get_val('ks.KS', indices=0), np.amax(prob.get_val('x')), tolerance=1e-8)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Computing Min and Max of KS**\n",
+    "\n",
+    "Because KSComp is a smooth max function, you can pass in any values to \"g\" and it will approximate the min and max. \n",
+    "\n",
+    "Here we set an upper limit which is lower than our \"g\" input and `lower_flag` to True to find the maximum value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "n = 50\n",
+    "prob = om.Problem()\n",
+    "model = prob.model\n",
+    "\n",
+    "prob.driver = om.ScipyOptimizeDriver()\n",
+    "\n",
+    "model.add_subsystem('comp', om.ExecComp('y = -3.0*x**2 + k',\n",
+    "                                        x=np.zeros((n, )),\n",
+    "                                        y=np.zeros((n, )),\n",
+    "                                        k=0.0), promotes_inputs=['x', 'k'])\n",
+    "\n",
+    "model.add_subsystem('ks', om.KSComp(width=n, upper=1.0))\n",
+    "\n",
+    "model.add_design_var('k', lower=-10, upper=10)\n",
+    "model.add_objective('k', scaler=-1)\n",
+    "model.ks.options['lower_flag'] = True\n",
```

Comment:
Set the lower_flag in the KSComp arguments instead, so that all options are together.